### DISCUSSION TITLE
What userid?

### DISCUSSION THREAD

**Discussion by gpu7** [VRSEN/agency-swarm#137] [Q&A] [answered]
Hi, when I run:

agency-swarm genesis --openai_key "key value"

is shows a USER: prompt.  What userid is it expecting here?

**Comment by nicko-ai** [comment, marked as answer]
Hey @gpu7, 

The "USER" prompt you're seeing when running agency-swarm genesis --openai_key "key value" is not asking for a userid or username. Instead, it's a conversational interface where you describe the agency you want to create.

  What to Enter:

  This is where you describe in natural language what kind of multi-agent system you want to build. For example:

  - "I want to create a customer support agency with agents that can handle tickets, escalate issues, and generate responses"
  - "Create a content creation agency with a writer, editor, and SEO specialist"
  - "I need a data analysis agency that can process CSV files, create visualizations, and generate reports"

  How It Works:

  1. The prompt "👤 USER:" indicates it's your turn to type
  2. Enter a description of your desired agency
  3. The GenesisCEO agent will process your request and may ask clarifying questions
  4. Once the requirements are clear, it will create the agency structure and all necessary agent files

  Special Commands:

  - Use @agent_name to direct messages to specific agents
  - Type exit to quit the genesis demo

  The system will guide you through the process interactively, so just describe what you want your agency to do and the Genesis system will help you create it!
